lib/hammer_cli_foreman/associating_commands.rb

```diff
@@ -229,23 +229,23 @@ class RemoveUsergroupCommand < HammerCLIForeman::RemoveAssociatedCommand
       end
     end
 
-    module ConfigTemplate
+    module ProvisioningTemplate
       extend CommandExtension
 
-      class AddConfigTemplateCommand < HammerCLIForeman::AddAssociatedCommand
-        associated_resource :config_templates
-        desc _("Associate a configuration template")
+      class AddProvisioningTemplateCommand < HammerCLIForeman::AddAssociatedCommand
+        associated_resource :provisioning_templates
+        desc _("Associate a Provisioning template")
```

Comment:
I don't think this needs capitalization:
```suggestion
        desc _("Associate a provisioning template")
```